javatests/arcs/android/storage/database/DatabaseImplTest.kt

```diff
@@ -404,6 +404,18 @@ class DatabaseImplTest {
     @Test
     fun insertAndGet_entity_newEntityWithPrimitiveFields() = runBlockingTest {
         val key = DummyStorageKey("key")
+
+        val inlineSchema = newSchema(
+            "inlineHash",
+            SchemaFields(
+                singletons = mapOf(
+                    "inlineText" to FieldType.Text,
+                    "inlineNumber" to FieldType.Number
+                ),
+                collections = mapOf()
```

Comment:
maybe add (just here is fine) a collection field, this pr already support them right?